src/transformers/testing_utils.py

```diff
@@ -648,6 +650,24 @@ def __repr__(self):
         return f"captured: {self.out}\n"
 
 
+@contextlib.contextmanager
+def LoggingLevel(level):
+    """
+
+    This is a context manager to temporarily change transformers modules logging level to the desired value and have it
+    restored to the original setting at the end of the scope. For example ::
+
+       with LoggingLevel(logging.INFO): # run some core function that calls logger.info() and it should be logged
+
```

Comment:
just noticed the styling formatter messed up:
```
with LoggingLevel(logging.INFO): 
    # run some core function that calls logger.info() and it should be logged
```

Reply:
hmm, if I put code there it still messed it up:

```
+       with LoggingLevel(logging.INFO): AutoModel.from_pretrained("gpt2")
```

what am I doing wrong? I though `::` indicates that it's code block in a docstring.

Reply:
Not sure. Can we put the comment above the with?

Reply:
I changed it to:
```

... Example ::

with LoggingLevel(logging.INFO): 
    AutoModel.from_pretrained("gpt2")
```

and it unwraps it.

Reply:
It comes from:
````
python utils/style_doc.py src/transformers docs/source --max_len 119
```

Reply:
I tried different indentation and adding paras after, to no avail.

Reply:
ok, the :: must not be in the first para - and then it works